SONAR-netCDF4/docs/crr341.adoc

```diff
@@ -466,54 +467,71 @@ noting that the order of the matrices is important, and is used:
+++++
+\begin{equation}
+R_{PCS->SCS} = R_z(h) \cdot R_y(p) \cdot R_z(r)
 \end{equation}
 ++++
 
-where latexmath:[M_z(h)] is the rotation about the _z_-axis by _h_ degrees, etc. The rotation symbols are chosen to reflect their common nautical names: _h_ for heading, _p_ for pitch, and _r_ for roll and used thus:
+
+where latexmath:[R_z(h)] is the rotation about the _z_-axis by _h_ degrees, etc. The rotation symbols are chosen to reflect their common nautical names: _h_ for heading, _p_ for pitch, and _r_ for roll and used thus:
 
 [stem]
 ++++
 \begin{eqnarray}
-M_x(r) =
+
+R_z(h) =
 \begin{bmatrix}
-1 & 0 & 0 \\
-0 & \cos(r) & -\sin(r) \\
-0 & \sin(r) & \cos(r)
+\cos(h) & -\sin(h) & 0 \\
+\sin(h) & \cos(h) & 0 \\
+0 & 0 & 1
 \end{bmatrix} \\
 
-M_y(p) =
+R_y(p) =
 \begin{bmatrix}
 \cos(p) & 0 & \sin(p) \\
 0 & 1 & 0 \\
 -\sin(p) & 0 & cos(p)
 \end{bmatrix} \\
 
-M_z(h) =
+R_x(r) =
 \begin{bmatrix}
-\cos(h) & -\sin(h) & 0 \\
-\sin(h) & \cos(h) & 0 \\
-0 & 0 & 1
+1 & 0 & 0 \\
+0 & \cos(r) & -\sin(r) \\
+0 & \sin(r) & \cos(r)
 \end{bmatrix}
 
 \end{eqnarray}
 ++++
 
+noting that the order of the matrices is important and should follow this order.
+
 To be clear, the right-hand rule when applied to roll, pitch, and heave means that:
 
 * looking along the positive _x_-axis, a positive rotation (roll) is clockwise (to starboard),
 * looking along the positive _y_-axis, a positive rotation (pitch) is clockwise (bow up),
-* looking along the positive _z_-axis, a positive rotation (yaw) is clockwise (to starboard).
+* looking along the positive _z_-axis, a positive rotation (heading) is clockwise (to starboard).
 
-The orientation of the platform is represented using the _z_–_y_’–_x_” Tait-Bryan intrinsic rotation convention (https://en.wikipedia.org/wiki/Euler_angles[en.wikipedia.org/wiki/Euler_angles]), corresponding to yaw, pitch, and roll, respectively. Intrinsic means that rotations about the _y_-axis are measured after any rotation about the _z_-axis, and rotations about the _x_-axis are measured after rotations about the _y_-axis (for comparison, extrinsic angles are applied relative to a fixed-platform orientation). This is the most used rotation convention in the maritime field, and the main effect is that roll is measured relative to the plane tilted by the pitch angle.
+The orientation of the platform is represented using the _z_–_y_’–_x_” Tait-Bryan intrinsic rotation convention (https://en.wikipedia.org/wiki/Euler_angles[en.wikipedia.org/wiki/Euler_angles]), corresponding to heading, pitch, and roll, respectively. Intrinsic means that rotations about the _y_-axis are measured after any rotation about the _z_-axis, and rotations about the _x_-axis are measured after rotations about the _y_-axis (for comparison, extrinsic angles are applied relative to a fixed-platform orientation). This is the most used rotation convention in the maritime field, and the main effect is that roll is measured relative to the plane tilted by the pitch angle.
```

Comment:
I thought we had moved to extrinsic rotations? Or perhaps I am confused...

Reply:
if I trust wikipedia, it is written that "the intrinsic rotations are known as: yaw, pitch and roll"

Reply:
If I trust wikipedia, it is written that "the intrinsic rotations are known as: yaw, pitch and roll"
So I guess we are intrinsic

Reply:
I understand it (again) now!